sdk/dashboard/Azure.ResourceManager.Dashboard/tests/TestsCase/GrafanaResourceOperationsTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Dashboard.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Dashboard.Tests.TestsCase
```

Comment:
Can we also test the `patch` method?

Reply:
Discussed offline. I will confirm with others, then add it if necessary